### DISCUSSION TITLE
[Extensions Update Checker] How do I prevent modified addons from being disabled when Firefox checks for addon updates?

### DISCUSSION THREAD

**Discussion by TFWol** [xiaoxiaoflood/firefox-scripts#323] [General Support]
I have a couple of addons I've modified to fix issues, but unlike the addons in this repo, they get disabled each time addons are checked for updates. The Extensions Update Checker script helped clue me in to the pattern.

Is there something special I should add to the extensions to prevent this?

**Comment by xiaoxiaoflood** [comment]
Being disabled? I also use a few mod versions for some extensions but I can't reproduce this issue. What I do is: when a new official version is released, I merge the changes in my fork (if applicable) then I add `.1-xiao` to version number to prevent further notifications until the next official release. So, for instance, if an extension receives official update to version `4.6`, I update my forked `.xpi` file to `4.6.1-xiao` (the `-xiao` is just so that I can easily see which extensions I'm using a personal fork) then install it manually.

**Reply by TFWol** [reply]
I like your version idea. I had changed the id, so that's prob what broke things.

The creator of the critical addon swung in at some point and fixed stuff, so I'll try just the version for the other one.

**Reply by TFWol** [reply]
Darn, it still gets disabled. I don't suppose there are files I shouldn't touch that would cause this are there?

**Reply by 117649** [reply]
I don't think modify Id can achieve what u want.

Nowadays addon on the Mozilla's site don't even include an id in .xpi . The id is later assigned after their signature is checked. The one of most important line in this report is to exclude our legacies from the sign check. You should get what you want by add some excluding codes there.
